docs/installation.md

```diff
@@ -129,67 +145,88 @@ From the root directory:
 
     $ vagrant up
 
-## From source or NPM
+## Manual install
 
-First of all, you have to get the source code. You can use NPM or clone our GIT repository.
+**Note:** we will assume that you want to launch Kuzzle and other services on the same host (localhost), but you can, of course, host kuzzle and any of its services on different hosts.
 
-* NPM
+### Prerequisites
 
-    ```
-    $ npm install kuzzle
-    ```
+* A service [Elasticsearch](https://www.elastic.co/products/elasticsearch) running on localhost:9200
+* A service [Redis](http://redis.io/) running on localhost:6379
+* A properly installed [nodeJs](https://nodejs.org/en/download/package-manager/) **version 4** or upper
+
+### Step one
 
-* GIT
+Retrieve the Kuzzle source code from the [GitHub repo](https://github.com/kuzzleio/kuzzle.git) by doing
 
-    ```
     $ git clone https://github.com/kuzzleio/kuzzle.git
     $ cd kuzzle
-    ```
 
-## Manual install
+then install the dependencies by doing:
 
-### Default
+    $ npm install
 
-Prerequisites:
+### Step two
 
-* A service [Elasticsearch](https://www.elastic.co/products/elasticsearch) running on localhost:9200
-* A service [Redis](http://redis.io/) running on localhost:6379
+Configure your environment. Kuzzle has been designed to be launched into a container, so the default hosts used to access to the ElasticSearch and Redis servers needs to be tweaked to hit the right hosts. If everything is hosted on localhost, you can use environment variable to overwrite default ones:
 
-```bash
-$ kuzzle start
-```
+    $ export READ_ENGINE_HOST=localhost:9200
+    $ export WRITE_ENGINE_HOST=localhost:9200
+    $ export CACHE_HOST=localhost
+    $ export CACHE_PORT=6379
 
-To get a list of available options, you can run:
 
-```bash
-$ kuzzle start -h
-```
+### Step three
 
-You may also start Kuzzle by using [PM2](https://github.com/Unitech/pm2):
+Install the default plugins, by doing:
 
-```bash
-$ pm2 start app-start.js
-```
+    $ ./bin/kuzzle.js install
 
 
-### Change external services hosts
+### Finally
+
+Start Kuzzle with default options (one server and one worker) by doing
```

Comment:
Remove `by doing`

The default `kuzzle start` runs 1 thread executing both a server and a worker. This is not necessarily the best way to run Kuzzle, except on a machine with very limited resources.

I would have written: 
```
Start a server instance:
$ bin/kuzzle.js start --server

And then as many worker instances as you want. At least 1 worker is required:
$ bin/kuzzle.js start --worker

For more information, you can execute:
$ bin/kuzzle.js start --help
```